docs-app/package.json

```diff
@@ -50,7 +50,6 @@
     "@tsconfig/ember": "^2.0.0",
     "@types/dompurify": "^3.0.0",
     "@types/ember": "^4.0.1",
-    "@types/ember-qunit": "^5.0.2",
```

Comment:
It appears we don't need these anymore with the latest version

Reply:
Sort of related: I think you can get rid of https://github.com/CrowdStrike/ember-toucan-core/blob/f1b73cd89c3f34319026ac7ed98d2304942c7a5d/docs-app/package.json#L71 (@types/ember__test-helpers) if you are using the most recent (ish) version of @ember/test-helpers

Rationale: https://www.npmjs.com/package/@types/ember__test-helpers

Reply:
Excellent catch! Just pushed the update